microprofile.jdt/org.eclipse.lsp4mp.jdt.core/plugin.xml

```diff
@@ -110,7 +110,11 @@
    <!-- Microprofile Config Static Property support -->
 
    <extension point="org.eclipse.lsp4mp.jdt.core.staticPropertyProviders">
-      <staticPropertyProvider resource="/static-properties/mp-config-metadata.json" />
+      <staticPropertyProvider resource="/static-properties/mp-config-metadata.json" >
+         <enabledWhen>
+            <typeOnClasspath type = "org.eclipse.microprofile.config.Config" />
```

Comment:
```suggestion
            <typeOnClasspath type="org.eclipse.microprofile.config.Config" />
```

If it works, it's fine. I'd just remove the spaces between the attribute declaration.